### PR TITLE
Update FOR_ANALYSTS.md

### PR DESCRIPTION
**Pull request by ccleung**
Tracker Ticket: 
https://quandl.atlassian.net/browse/AP-1691

## Description:  
* README section for analyst `quandl.bulkdownload`
* fix `get_table` call for analysts
* fix databases metadata call for developers

### CODE REVIEW

File: FOR_ANALYSTS.md [hunk 1]
```diff
@@ -48,6 +48,31 @@ The following additional parameters can be specified for a dataset call:
 
 For more information on how to use and manipulate the resulting data see the [pandas documentation](http://pandas.pydata.org/).
 
+#### Download Entire Database (Bulk Download)
+
+You can download all the data of a database in a single call. The following will download the entire EOD database as a zip file to your current working directory:
```
Comment (junos) [inline FOR_ANALYSTS.md]: should be `all data of a` instead of `all the data of a`?

Comment (junos) [inline FOR_ANALYSTS.md]: `The following` should be `The following script`?

Comment (ccleung) [inline FOR_ANALYSTS.md]: Im not sure what the best wording may be, but i copied "all the data" from here: https://www.quandl.com/docs/api#entire-database maybe it should be "all the data in a database..."

**Comment by junos** [comment]
:+1:  pending on CI

**Pull request merged**